lib/i18n/backend/active_record.rb

```diff
@@ -30,103 +27,111 @@ def initialize(*args)
         reload!
       end
 
-      def available_locales
-        Translation.available_locales
-      rescue ::ActiveRecord::StatementInvalid
-        []
-      end
+      module Implementation
+        include Base, Flatten
```

Comment:
Rubocop won't like this
```suggestion
        include Base
        include Flatten
```

Reply:
Done 👍🏼